x-pack/plugins/timelines/server/search_strategy/timeline/utils/constants.ts

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ALERT_RULE_CONSUMER } from '@kbn/rule-data-utils';
+// import { CTI_ROW_RENDERER_FIELDS } from '../../../../../../common/cti/constants';
```

Comment:
nit: not sure whether or not this commented `import` should stay, because it's related to the `TODO` below:

```
// TODO: share with security_solution/common/cti/constants.ts
```

Reply:
I removed it from here, but added this comment as subpoint in the tech debt doc here for de-duping of files https://github.com/elastic/security-team/issues/2441